### PR TITLE
ad stilt availability table

### PR DESCRIPTION
**Pull request by altix**
Tested on jupyterLab. 
- new filter: stiltstation.find( project = 'icos')
- new outfmt: stiltstation.find( outfmt = 'avail')   
the output format for the availability table is a dataframe.

### CODE REVIEW

**Review by ukarst — APPROVED** [review]
OK, availability table looks good, filter for icos stations will automatically improve after #64 is done

File: icoscp/stilt/stiltstation.py [hunk 1]
```diff
@@ -504,11 +559,14 @@ def find(**kwargs):
         Remember, that only year and month is checked.
     Example: station.find(dates=['2020-01-01', '2020/05/23'])
 
+    project STR 'icos':
+        This will only return stilt stations that are ICOS stations.
+
```
Comment (claudiodonofrio) [inline icoscp/stilt/stiltstation.py:564]: please add example, to be consistent with the other help entries.

**Review by claudiodonofrio — COMMENTED** [review]
hi please add an example to the help entry for project=...
and don't forget to update the help docs accordingly with the new features

File: icoscp/stilt/stiltstation.py (hunk 1 above)
Comment (altix) [reply icoscp/stilt/stiltstation.py:566]: Added example

**Review by claudiodonofrio — APPROVED** [review]

**Review by ZogopZ — APPROVED** [review]

**Pull request merged**